appium/webdriver/extensions/log_event.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium import webdriver
+
+from ..mobilecommand import MobileCommand as Command
+
+
+class LogEvent(webdriver.Remote):
+
+    @property
+    def events(self):
```

Comment:
https://github.com/appium/appium-base-driver/blob/master/lib/protocol/routes.js#L589-L594
`/events` route accept `type` argument as option.
It hasn't been implemented yet, but this `events` should not be property.

You can rename here as `get_events(self, type = None)` to keep another `events` as `session_capability['events']` as backword compatibility as https://github.com/appium/python-client/pull/469#issuecomment-552186783

Reply:
@KazuCocoa 
Thanks, I didn't understand for details, I will keep original `events` and create `get_events` as another one.